src/index.js

```diff
@@ -26,6 +26,18 @@ const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));
 
 export async function run() {
   try {
+    // custom working directory
+    const workingDirectoryInput = core.getInput('working-directory');
+    if (workingDirectoryInput) {
+      core.info(`Starting with custom working directory: ${workingDirectoryInput}`);
```

Comment:
This new `core.info()` is unexpected by the tests and therefore causes [this failure](https://github.com/mxschmitt/action-tmate/pull/157/checks#step:5:51). I would suggest to either drop the `core.info()` call, or to adjust the tests.